build-tools/automation/azure-pipelines.yaml

```
@@ -16,7 +16,7 @@ pr:
 
 # Global variables
 variables:
-  DotNetCoreVersion: 3.1.300
+  DotNetCoreVersion: 5.0.100-rc.2.20479.15
```

You might try using .NET 5.0 GA that came out:
```suggestion
  DotNetCoreVersion: 5.0.100
```